src/hooks/useDomainInfo.tsx

```diff
@@ -110,6 +111,10 @@ export function buildDomainInfoQuery({
         process: new AOProcess({
           processId,
           ao: antAo,
+          logger: new SentryLogger({
+            component: 'ANT',
```

Comment:
Why ANT as opposed to something like ArNS-App?

Reply:
I read more about how these are used in Sentry and I think you're free to choose these however is meaningful for grouping within the project. ANT is fine by me if you think it makes sense organizationally.